slothy/targets/arm_v81m/arch_v81m.py

```diff
@@ -2250,6 +2278,12 @@ def make(cls, src):
         return obj
 
 
+class vmsr(MVEInstruction):
+    pattern = "vmsr <Rs>, <Rn>"
```

Comment:
This doesn't do what you think it does. Rs will be seen as a symbolic register and replaced by an `r` register, so that doesn't work. 
You will have to hardcode the possible flags. i.e., use 
```
pattern = "vmsr FPSCR, <Rn>"
```

But maybe we can avoid using `vmsr` altogether. let me check .